h/auth/util.py

```diff
@@ -14,17 +11,10 @@ def client_authority(request):
     """
     Return the authority associated with an authenticated auth_client or None.
 
-    Once a request with an auth_client is authenticated, a principal is set
-    indicating the auth_client's verified authority
-
-    see :func:`~h.auth.util.principals_for_auth_client` for more details on
-    principals applied when auth_clients are authenticated
-
     :rtype: str or None
     """
-    for principal in request.effective_principals:
-        match = re.match(r"^client_authority:(.+)$", principal)
-        if match and match.group(1):
-            return match.group(1)
+    # This function is kind of dumb and should be removed...
```

Comment:
```suggestion
```

I think we should avoid `TODO` comments in the code: they're not effective at causing things to get done, and often become out-of-date and misleading. We should avoid comments that insult the code too